inc/tag.class.php

```diff
@@ -440,6 +440,62 @@ static function preItemForm($params = []) {
       }
    }
 
+   /**
+    * Display the current tags before the Kanban item content.
+    *
+    * @param  array $params should contains theses keys:
+    *                          - itemtype The item type
+    *                          - items_id The item's id
+    *                          - content postKanbanContent content
+    * @return array Array of params passed in in addition to the new content.
+    */
+   static function preKanbanContent($params = []) {
+      global $DB;
+
+      if (!Session::haveRight(PluginTagTag::$rightname, READ)) {
+         return null;
+      }
+
+      if (isset($params['itemtype']) && isset($params['items_id'])) {
+         if (!isset($params['content'])) {
+            $params['content'] = "";
+         }
+         $iterator = $DB->request([
+            'SELECT'    => [
+               'name',
+               'comment',
+               'color'
+            ],
+            'FROM'      => PluginTagTagItem::getTable(),
+            'LEFT JOIN' => [
+               PluginTagTag::getTable() => [
+                  'FKEY'   => [
+                     PluginTagTag::getTable()      => 'id',
+                     PluginTagTagItem::getTable()  => 'plugin_tag_tags_id'
+                  ]
+               ]
+            ],
+            'WHERE'     => [
+               'itemtype'  => $params['itemtype'],
+               'items_id'  => $params['items_id']
+            ]
+         ]);
+
+         $content = "<div style='display: flex; flex-wrap: wrap;'>";
+         while ($data = $iterator->next()) {
+            $title = htmlentities($data['comment']);
+            $name = htmlentities($data['name']);
+            $textcolor = idealTextColor($data['color']);
+            $style = "background-color: {$data['color']}; color: {$textcolor}; padding: 2px; border-radius: 5px; margin-top: 5px;";
```

Comment:
instead inline css, i think you can use `.tag_choice` class already used in other glpi view (more consistent display).
Keep only background and foreground colors.
Ex: 
![image](https://user-images.githubusercontent.com/418844/65312297-7ec41a80-db92-11e9-941e-48ff0a5fd19c.png)